src/layouts/dashboard/sidebar.tsx

```diff
@@ -29,15 +29,17 @@ const Sidebar = ({ onDrawer }: SidebarProps) => {
     route,
     drawer,
     menuItems,
-    vaultAssets,
     vaultRequest,
     transactionListRequest: {
       pendingTransactions,
       pendingSignerTransactionsLength,
     },
   } = useSidebar();
```

Comment:
this react component needs use context with `useVaultDetails`

here, import by context `useVaultInfosContext`

Reply:
Done